Automattic/woocommerce-payments
# Resolve collision between WooPayments header and Woo Express survey banner

Fixes #8093

#### Changes proposed in this Pull Request
This PR removes the JITM notices on onboarding page to prevent any distractions during onboarding.

#### Testing instructions
- Start Woo Express trial.
- Make sure that you don't finish Woo Express survey. It should be on the top of the page.
- Once the initial setup is finished, click on the action item to start with setting up WooPayments.
- Upload WooPayment plugin with current change.
- Use the following [gist].(https://gist.githubusercontent.com/ilyasfoo/a2901efacd62f69be737778ba7721e69/raw/5e18c6fc82f52ecf4c552cec5bc3b092540338dd/jQueryJITM) to force show the survey banner.
- Verify on the wizard page the survey banner is hidden.

*

-------------------


**Post merge**
